components/lib/divider/Divider.spec.js

```diff
@@ -0,0 +1,85 @@
+import '@testing-library/jest-dom';
+import { render } from '@testing-library/react';
+import { Divider } from './Divider';
+
+describe('Divider', () => {
+    test('when tpe has not any property it returns with default class', () => {
+        // Arrange
+        const { container } = render(<Divider />);
+
+        // Act + Assert
+        expect(container.getElementsByClassName('p-divider p-component p-divider-horizontal').length).toBe(1);
```

Comment:
A better way to test for this is
```js
 expect(container).toHaveClass('p-divider p-component p-divider-horizontal');
```

Reply:
I would make this change throughout this whole test

Reply:
Have you considered using https://jestjs.io/docs/snapshot-testing ?
I think it is more convenient, as the test code is separate from the expected output, so you do not need to maintain these classes in the test files. When something changes, it is usually more clear what the changes are when observing them in the snapshot.

Reply:
Hi. I don't think it's a UI control. It tests the effect of certain logic on the UI and gets help from classes. For this reason, I could not associate it with snapshot tests. But if you'd like to send a sample, we'd be happy to review it.

Reply:
OK, I will send you an example what I mean. It is capturing the generated markup and making sure it does not change unexpectedly.

Reply:
Check here: #3579